ViMac-Swift/Modes/HintsViewController.swift

```diff
@@ -70,7 +70,15 @@ class HintsViewController: NSViewController {
     func renderHint(_ hint: Hint) -> HintView? {
         let view = HintView(associatedElement: hint.element, hintTextSize: CGFloat(textSize), hintText: hint.text, typedHintText: "")
         guard let elementFrame = self.elementFrame(hint.element) else { return nil }
-        let elementCenter: NSPoint = GeometryUtils.center(elementFrame)
+        
+        var elementCenter: NSPoint
+        if hint.element.role == "AXLink" {
+            // It's likely that the hint text size chosen by the user is roughly equivalent to their viewing text size, hence textSize being used as the offset.
```

Comment:
I think its more intuitive to change the computation of `hintOrigin` instead of changing the value of `elementCenter`.

Reply:
Currently the hints will be displayed on bottom-left, but clicking will still be done at the center. The hint clicking implementation in `HintModeController#performHintAction` needs to be changed as well.

Reply:
I've made the changes here. Could you pull them into this branch?

https://github.com/dexterleng/vimac/compare/axlink-fix

Reply:
I've merged the changes into this PR and also removed the `corner` function.